app/views/services/edit.html.erb

```diff
@@ -35,6 +35,10 @@
               method: :delete, class: 'destructive delete' %>
             <% end %>
           </li>
+          <li data-action="edit">
+            <%= link_to 'Add branching',
+              new_branches_path(service.service_id, page.uuid) %>
+          </li>
```

Comment:
can we take this out please. it's on the `testable-add-branching-link` branch

Reply:
Done